vardautomation/tooling.py

```diff
@@ -1145,7 +1145,7 @@ def __init__(self, binary: AnyPath, settings: Union[AnyPath, List[str], Dict[str
         self.progress_update = progress_update
         super().__init__(binary, settings)
 
-    def run_enc(self, clip: vs.VideoNode, file: Optional[FileInfo], *, y4m: bool = True) -> None:
+    def run_enc(self, clip: vs.VideoNode, file: Optional[FileInfo], *, y4m: bool = True, prefetch: Optional[int] = None) -> None:
```

Comment:
What about adding `prefetch` as class attribute?

Thus:
```py
encoder = VideoEncoder(...)
encoder.prefetch = some_value

encoder.run_enc(...)
```
We should move `y4m` too in this case to remain consistent.

Reply:
Sure, that works too, and i believe going this way, adding other things would be more practical in the future